Dockerfile

```diff
@@ -10,13 +10,43 @@ RUN apt-get update && \
       apt-get install -y libgtk2.0-0 libgconf-2-4 \
```

Comment:
It is not clear to me why this is not merging clean - but this is the Dockerfile I was able to produce that made git happy:

```
ARG NODE_VERSION=8.9.1
FROM node:${NODE_VERSION}

# Add our user and group first to make sure their IDs get assigned consistently,
# regardless of whatever dependencies get added.
RUN groupadd --system nightmare && useradd --system --create-home --gid nightmare nightmare

# Installing the pre-required packages and libraries for electron & Nightmare
RUN apt-get update && \
      apt-get install -y libgtk2.0-0 libgconf-2-4 \
      libasound2 libxtst6 libxss1 libnss3 xvfb

# Install latest chrome dev package.
# Note: this installs the necessary libs to make the bundled version of Chromium that Puppeteer
# installs, work.
RUN apt-get update && apt-get install -y wget --no-install-recommends \
    && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
    && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
    && apt-get update \
    && apt-get install -y google-chrome-unstable \
      --no-install-recommends \
    && rm -rf /var/lib/apt/lists/* \
    && apt-get purge --auto-remove -y curl \
    && rm -rf /src/*.deb

# Uncomment to skip the chromium download when installing puppeteer. If you do,
# you'll need to launch puppeteer with:
#     browser.launch({executablePath: 'google-chrome-unstable'})
# ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD true

# Install puppeteer so it's available in the container.
RUN yarn add puppeteer

# Add pptr user.
RUN groupadd -r pptruser && useradd -r -g pptruser -G audio,video pptruser \
    && mkdir -p /home/pptruser/Downloads \
    && chown -R pptruser:pptruser /home/pptruser \
    && chown -R pptruser:pptruser /home/pptruser \
    && chown -R pptruser:pptruser /node_modules

RUN mkdir /codecept && chown -R pptruser:pptruser /codecept

WORKDIR /tmp
COPY package.json /tmp/

# Install packages
RUN npm install --loglevel=warn

#RUN mkdir /home/codecept
WORKDIR /codecept

COPY . /codecept

RUN cp -a /tmp/node_modules /codecept/

# Allow to pass argument to codecept run via env variable
ENV CODECEPT_ARGS=""

# Set HOST ENV variable for Selenium Server
ENV HOST=selenium

# Run user as non privileged.
# USER pptruser

# Set the entrypoint for Nightmare
ENTRYPOINT ["docker/entrypoint"]

# Run tests
CMD ["bash", "docker/run.sh"]
```